jdk_21_maven/cs/web/triangle-ssr/src/main/java/com/example/springmvcdocker/FormController.java

```diff
@@ -0,0 +1,62 @@
+package com.example.springmvcdocker;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class FormController {
+
+    @GetMapping("/index")
+    public String showForm() {
```

Comment:
why this endpoint?  wouldn't make more sense that root "/" is mapped into "index.html"? ie `@GetMapping("/") ... return "index.html"`?